### PR TITLE
Un-Embedding the link.xml 

### PR DESCRIPTION
**Pull request by bitsandfoxes**
What happened so far:

1. Initially link.xml was inside Resources/Sentry and got shipped with the build
2. Wanted to move it somewhere else. Can't be inside /Packages because the managed code stripper doesn't look there.
3. Embedded it into the assembly. But that doesn't work with a wide range of Unity versions.
4. Placing it now in Assets/Plugins/Sentry

### CODE REVIEW

File: src/Sentry.Unity.Editor/SentryPreprocessBuild.cs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+using System.IO;
```
Comment (semuserable) [inline src/Sentry.Unity.Editor/SentryPreprocessBuild.cs:1]: discard?

File: src/Sentry.Unity.Editor/SentryWindow.cs [hunk 2]
```diff
@@ -175,6 +179,33 @@ private static void TryCreateSentryFolder()
...
+                AssetDatabase.CreateFolder("Assets/Plugins", "Sentry");
+            }
+
+            if (!AssetDatabase.IsValidFolder(LinkXmlPath))
+            {
+                using var fileStream = File.Create(LinkXmlPath);
+                using var resourceStream = GetType().Assembly.GetManifestResourceStream("Sentry.Unity.Editor.Resources.link.xml");
+                resourceStream.CopyTo(fileStream);
+
+                fileStream.Close();
```
Comment (semuserable) [inline src/Sentry.Unity.Editor/SentryWindow.cs:203]: This seems redundant as you already have `using` on `File.Create`.

File: src/Sentry.Unity.Editor/SentryWindow.cs [hunk 3]
```diff
@@ -175,6 +179,33 @@ private static void TryCreateSentryFolder()
...
+                resourceStream.Flush();
```
Comment (semuserable) [inline src/Sentry.Unity.Editor/SentryWindow.cs:204]: Same as above.

**Review by semuserable — APPROVED** [review]

**Review by bruno-garcia — APPROVED** [review]
Thanks!

**Pull request merged**